doc/api/topics/index_-_ok_-_without_params.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Is the filename autogenerated? Any chance we can remove the `-`?

Reply:
@JulianPasquale  That's because the name of the file is autogenerated based on this description:
https://github.com/LeandroSoria2/TargetApi/blob/d827a8f9ca4fcff6268fe190b954ed621d4ca8fe/spec/acceptance/topics_spec.rb#L19

Reply:
Then I'd suggest to use a different example description 🙂

Reply:
As a side note, that'll also change this title

<img width="833" alt="imagen" src="https://user-images.githubusercontent.com/80283482/180273368-d5af0b60-4c93-41da-a604-c1aa5da7bb9b.png">

Reply:
I agree with @JulianPasquale. I'd use a more natural name, like `Get all topics` or similar. Not a blocker.